contract/apps.py

```diff
@@ -1,5 +1,43 @@
 from django.apps import AppConfig
 
 
```

Comment:
add some perms, later I will add another ones in "OFS-154: create contract role for admin"